config-options/xml-configuration/README.adoc

```diff
@@ -1,30 +1,48 @@
 = XML Configuration
 
 This example demonstrates how to use XML configuration instead of the java config-api,
- to configure WildFly Swarm.
+to configure WildFly Swarm. This might be useful, when you are migrating from a traditional
+WildFly deployment and you have your `standalone.xml` file ready.
 
 
 == Referencing standalone.xml
 
-
 To reference a `standalone.xml` it can be part of your project, where it's picked up automatically,
 or you can pass it to WildFly Swarm with a `-c` parameter:
 
-[source,xml]
+[source,sh]
 ----
-include::pom.xml[lines=39..45]
+cd xml-configuration
+java -jar target/example-xml-config-swarm.jar -c ../standalone.xml
 ----
 
-See: link:pom.xml#L42[pom.xml]
-
 == Subsystem configuration in XML
 
 You can then have any subsystem configuration external in `standalone.xml`.
 These configuration will then be matched with the corresponding fractions automatically.
 
+In in our example link:../standalone.xml#L130-L147[standalone.xml] file, we configure a custom datasource
+called `CustomDS`:
+
 [source,xml]
 ----
-include::../standalone.xml[lines=130..147]
```

Comment:
Why was the include removed?

It may only work locally, but that's why we have the link below it to the actual location.

This ensures the README doesn't become out of date with the code